admin/app/routes/authenticated/complementary-certifications/complementary-certification/details.js

```diff
@@ -11,4 +11,10 @@ export default class ComplementaryCertificationDetailsRoute extends Route {
   async model() {
     return this.modelFor('authenticated.complementary-certifications.complementary-certification');
   }
+
+  resetController(controller, isExiting) {
+    if (isExiting) {
+      controller._targetProfileId = null;
```

Comment:
On touche ici à un attribut explicitement privé du controlleur. On pourrait plutôt lui demander gentiment de le faire:

```suggestion
      controller.resetTargetProfileId();
```